python/tank/util/qt_importer.py

```diff
@@ -508,6 +503,9 @@ def _import_modules(self, interface_version_requested):
             except ImportError:
                 pass
 
+            # TODO migrate qt base from Qt4 interface to Qt6 will require patching Qt5 as Qt6
+            logger.debug("Qt6 interface not implemented for Qt5")
```

Comment:
(This comment is more for the general deprecation of QT4)

This `TODO` is only needed if we want to fully support the QT6 interface. It may not be necessary if when deprecating QT4, we update to QT5 as the default interface -- for example, currently QT4 is the default interface, so we make sure that all versions of Qt are patched to provide a QT4 interface, but we do not provide all patches for other interfaces.

If we do update to QT5 as the default interface, then we'll need to add a PySide6 patch for QT5 interface:
```
elif interface_version_requested == self.QT5:

    ...

    try:
        pyside6 = self._import_pyside6_aas_pyside2
        logger.debug("Imported PySide6")
        return pyside6
    except ImportError:
        pass
```